frontend/src/main/pages/Shift/ShiftIndexPage.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useBackend } from 'main/utils/useBackend';
+import BasicLayout from 'main/layouts/BasicLayout/BasicLayout';
+import ShiftTable from 'main/components/Shift/ShiftTable';
+import { Button } from 'react-bootstrap';
+import { useCurrentUser, hasRole } from 'main/utils/currentUser';
+
+export default function ShiftIndexPage() {
+
+    const currentUser = useCurrentUser();
+
+    const { data: shift, error: _error, status: _status } =
+    useBackend(
+        // Stryker disable all : don't test internal caching of React Query
+        ["/api/shift/all"],
+        { method: "GET", url: "/api/shift/all" },
+        []
+        // Stryker restore all
+    );
+
+    const createButton = () => {
+        if (hasRole(currentUser, "ROLE_ADMIN")) {
+            return (
+                <Button
+                    variant="primary"
+                    href="/shift/create"
+                    style={{ float: "right" }}
+                >
+                    Create Shift
+                </Button>
+            );
+        }
+    };
+    
+    return (
+        <BasicLayout>
+            <div className="pt-2">
+                {createButton()}
```

Comment:
```suggestion
           ( hasRole(currentUser, "ROLE_ADMIN")) &&
                <Button
                    variant="primary"
                    href="/shift/create"
                    style={{ float: "right" }}
                >
                    Create Shift
                </Button>
         );
```

Reply:
@chriscerie Is the suggestion above what you are asking @lawrencewang30 to do?

We are in the final moments of the quarter, and I'm trying to help the teams wrap things up. I appreciate your suggestions, but we may need to just move forward on this and refactor it later.   If you can answer quickly, we may be able to incorporate your suggestion; otherwise, maybe it's something we'll do later on .

Reply:
Yes, but without the semicolon and wrapped in brackets